values.yaml

```diff
@@ -103,6 +103,10 @@ global:
   ssl:
     enabled: false
     mode: "prefer"
+    # we need seperate mode for grafana: 
+    # it supports only disable/require/verify-ca/verify-full
+    grafana:
+      sslmode: "disable"
```

Comment:
Why did we opt to default `disable` here?

Reply:
because default option ssl disabled, but we can change to `require`, so if somebody change ssl: enable globally. It will work. I'll fix.